virttest/utils_test/libvirt.py

```diff
@@ -2492,16 +2492,21 @@ def get_vm_device(vmxml, dev_tag, index=0):
     return (dev_obj, xml_devices)
 
 
-def add_vm_device(vmxml, new_device, virsh_instance=virsh):
+def add_vm_device(vmxml, new_device, virsh_instance=virsh, need_sync=True):
```

Comment:
For the name of param, I'd like to suggest `sync` or `sync_vm`, just suggesting

Reply:
Updated, thanks